index.d.ts

```diff
@@ -1,69 +0,0 @@
-// Type definitions for core-decorators.js 0.19
```

Comment:
no longer need this because TypeScript is generating individual `.d.ts` files per source file.